inc/dbmysql.class.php

```diff
@@ -1560,6 +1560,13 @@ public function inTransaction() {
     * @since 9.5.0
     */
    public function areTimezonesAvailable(string &$msg = '') {
+      $cache = Config::getCache('cache_db');
```

Comment:
you can get the cache by the global `$CACHE`, and i think it's better for future replacement by a DI to have the same syntax everywhere

Reply:
`Config::getCache` is deprecated since https://github.com/glpi-project/glpi/commit/b0f65de. You have to use the global `$GLPI_CACHE` instead.

Reply:
I tried to use `$GLPI_CACHE` first but it breaks the console: https://github.com/glpi-project/glpi/runs/3494534869
Looks like `$GLPI_CACHE` doesn't implement the null object pattern and can't be trusted.

Reply:
In 9.5, `$GLPI_CACHE` instanciation requires a DB instance (as cache configuration is stored inside DB), so if DB instanciation requires a cache instance, then you cannot use `$GLPI_CACHE` without creating a circular dependency.
I doubt there is a better solution in 9.5 branch.

My thought is that, maybe, in master version, we should try to find a solution to move a `use_timezones` boolean into the DB configuration file, as we do for the `use_utf8mb4`, in order to remove all these checks. Defining if timezones can and must be used is something that will not be done frequently, and which depends on system configuration that is unlikely to change, so I guess we could do these checks and set a boolean during installation/update phase and update it during timestamps migration.